### PR TITLE
Fix `TextureRegionEditor::_pan_callback` deferred called with only one argument

### LINKED ISSUE #87703: Error about deferred method call is printed when opening the TextureRegion editor
### Tested versions

- Reproducible in: 4.3.dev.0bcc0e92b
- Not reproducible in: 4.2.1.stable

### System information

Godot v4.3.dev (0bcc0e92b) - Fedora Linux 39 (KDE Plasma) - X11 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 4090 (nvidia; 545.29.06) - 13th Gen Intel(R) Core(TM) i9-13900K (32 Threads)

### Issue description

```
ERROR: Error calling deferred method: 'TextureRegionEditor::TextureRegionEditor::_pan_callback': Method expected 2 arguments, but called with 1.
   at: _call_function (core/object/message_queue.cpp:223)
```

### Steps to reproduce

Create a Sprite2D node with a GradientTexture2D, enable **Region** on it and click **Edit Region** in the inspector.

### Minimal reproduction project (MRP)

N/A

### PR DESCRIPTION
**Pull request by jsjtxietian**
Fixes https://github.com/godotengine/godot/issues/87703

### CODE REVIEW

**Review by Calinou — APPROVED** [review]
Tested locally, it works as expected.

File: editor/plugins/texture_region_editor_plugin.cpp [hunk 1]
```diff
@@ -226,7 +226,7 @@ void TextureRegionEditor::_texture_overlay_draw() {
 		hscroll->set_value((hscroll->get_min() + hscroll->get_max() - hscroll->get_page()) / 2);
 		vscroll->set_value((vscroll->get_min() + vscroll->get_max() - vscroll->get_page()) / 2);
 		// This ensures that the view is updated correctly.
-		callable_mp(this, &TextureRegionEditor::_pan_callback).bind(Vector2(1, 0)).call_deferred();
+		callable_mp(this, &TextureRegionEditor::_pan_callback).bind(Vector2(1, 0), Ref<InputEvent>()).call_deferred();
 		callable_mp(this, &TextureRegionEditor::_scroll_changed).bind(0.0).call_deferred();
```
Comment (KoBeWi) [inline editor/plugins/texture_region_editor_plugin.cpp:230]: tbh the code should be changed to this:
```suggestion
		callable_mp(this, &TextureRegionEditor::_pan_callback).call_deferred(Vector2(1, 0), Ref<InputEvent>());
		callable_mp(this, &TextureRegionEditor::_scroll_changed).call_deferred(0.0);
```
`bind()` doesn't make sense in this context (although both will work, so it's stylistic change).

**Review by KoBeWi — APPROVED** [review]
Didn't test, but the fix is correct.

File: editor/plugins/texture_region_editor_plugin.cpp (hunk 1 above)
Comment (jsjtxietian) [reply editor/plugins/texture_region_editor_plugin.cpp:230]: Thanks, I will apply your suggestion today.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**